libdisni/src/verbs/com_ibm_disni_verbs_impl_NativeDispatcher.cpp

```diff
@@ -823,6 +892,59 @@ Java_com_ibm_disni_verbs_impl_NativeDispatcher__1queryOdpSupport(JNIEnv *env,
   return ret;
 }
 
+/*
+ * Class:     com_ibm_disni_verbs_impl_NativeDispatcher
+ * Method:    _queryMaxResponderResources
+ * Signature: (J)I
+ */
+JNIEXPORT jint JNICALL
+Java_com_ibm_disni_verbs_impl_NativeDispatcher__1queryMaxResponderResources(
```

Comment:
That is not what I had in mind. I would like to be able to get all device attributes in Java. Otherwise we will be adding new JNI functions for every member of the struct along the way.

Reply:
Sure. it was just more work.